helmclient.go

```diff
@@ -286,6 +295,7 @@ func (c *Client) EnsureTillerInstalled(ctx context.Context) error {
 		MaxHistory:                   defaultMaxHistory,
 		Namespace:                    c.tillerNamespace,
 		ServiceAccount:               tillerPodName,
+		Values:                       values,
```

Comment:
This is all that changes, very straight forward just passing this through to the helm API.